Detectors/Upgrades/ALICE3/FCT/simulation/src/FCTLayer.cxx

```diff
@@ -0,0 +1,202 @@
+    TGeoTube* sensor = new TGeoTube(mInnerRadius, mOuterRadius, mChipThickness / 2);
+    TGeoTube* chip = new TGeoTube(mInnerRadius, mOuterRadius, mChipThickness / 2);
+    TGeoTube* layer = new TGeoTube(mInnerRadius, mOuterRadius, mChipThickness / 2);
+
+    TGeoMedium* medSi = gGeoManager->GetMedium("FCT_SI$");
+    TGeoMedium* medAir = gGeoManager->GetMedium("FCT_AIR$");
+
+    TGeoVolume* sensVol = new TGeoVolume(sensName.c_str(), sensor, medSi);
+    TGeoVolume* chipVol = new TGeoVolume(chipName.c_str(), chip, medSi);
+    TGeoVolume* layerVol = new TGeoVolume(mLayerName.c_str(), layer, medAir);
+
+    LOG(info) << "Inserting " << sensVol->GetName() << " inside " << chipVol->GetName();
+    chipVol->AddNode(sensVol, 1, nullptr);
+
+    LOG(info) << "Inserting " << chipVol->GetName() << " inside " << layerVol->GetName();
+    layerVol->AddNode(chipVol, 1, nullptr);
+
+    // Finally put everything in the mother volume
+    auto* FwdDiskRotation = new TGeoRotation("FwdDiskRotation", 0, 0, 180);
+    auto* FwdDiskCombiTrans = new TGeoCombiTrans(0, 0, mZ, FwdDiskRotation);
+
+    LOG(info) << "Inserting " << layerVol->GetName() << " inside " << motherVolume->GetName();
+    motherVolume->AddNode(layerVol, 1, FwdDiskCombiTrans);
+
+    return;
+  }
+}
+
+void FCTLayer::createSquareLayer(TGeoVolume* motherVolume)
+{
+  if (mLayerNumber >= 0) {
+
+    LOG(info) << "Constructing a layer and adding it to the motherVolume";
+
+    std::string chipName = o2::fct::GeometryTGeo::getFCTChipPattern() + std::to_string(mLayerNumber),
+                sensName = Form("%s_%d_%d", GeometryTGeo::getFCTSensorPattern(), mDirection, mLayerNumber);
+    TGeoBBox* sensorBox = new TGeoBBox("SensorBox", mSideLength, mSideLength, mChipThickness / 2);
+    TGeoBBox* chipBox = new TGeoBBox("ChipBox", mSideLength, mSideLength, mChipThickness / 2);
+    TGeoBBox* layerBox = new TGeoBBox("LayerBox", mSideLength, mSideLength, mChipThickness / 2);
+
+    TGeoTube* sensorCutout = new TGeoTube("SensorTube", 0., mInnerRadius, mChipThickness / 2);
+    TGeoTube* chipCutout = new TGeoTube("ChipTube", 0., mInnerRadius, mChipThickness / 2);
+    TGeoTube* layerCutout = new TGeoTube("LayerTube", 0., mInnerRadius, mChipThickness / 2);
+
+    TGeoCompositeShape* SensorComp = new TGeoCompositeShape("SensorComp", "SensorBox - SensorTube");
+    TGeoCompositeShape* ChipComp = new TGeoCompositeShape("SensorComp", "ChipBox - ChipTube");
+    TGeoCompositeShape* LayerComp = new TGeoCompositeShape("SensorComp", "LayerBox - LayerTube");
+
+    TGeoMedium* medSi = gGeoManager->GetMedium("FCT_SI$");
+    TGeoMedium* medAir = gGeoManager->GetMedium("FCT_AIR$");
+
+    TGeoVolume* sensVol = new TGeoVolume(sensName.c_str(), SensorComp, medSi);
+    TGeoVolume* chipVol = new TGeoVolume(chipName.c_str(), ChipComp, medSi);
+    TGeoVolume* layerVol = new TGeoVolume(mLayerName.c_str(), LayerComp, medAir);
+
+    LOG(info) << "Inserting " << sensVol->GetName() << " inside " << chipVol->GetName();
```

Comment:
actually there are a lot of `LOG(info)` in this file and others. Could some of them be replaced by a `LOG(debug)`?